www/include/eventLogs/viewLog.php

```diff
@@ -68,54 +68,82 @@
 $tpl = new Smarty();
 $tpl = initSmartyTpl("./include/eventLogs/template", $tpl);
 
-$engine = 'false';
-if (isset($_GET["engine"]) && $_GET["engine"] == 'true') {
-    $engine = 'true';
-}
 
-$output = "";
-if (isset($_GET["output"])) {
-    $output = $_GET["output"];
-}
 
-$openid = '0';
-if (isset($_GET["openid"])) {
-    $openid = $_GET["openid"];
+/**
+ * filteredArrayId
+ *
+ * @param  mixed $ids
```

Comment:
What kind of values are in this array?
You can type this array directly using int[].
```suggestion
 * @param  int[] $ids
```